pkg/jobrunaggregator/jobrunaggregatorapi/types_row_testrun.go

```diff
@@ -0,0 +1,97 @@
+package jobrunaggregatorapi
+
+import (
+	"strings"
+
+	"cloud.google.com/go/bigquery"
+
+	prowv1 "k8s.io/test-infra/prow/apis/prowjobs/v1"
+
+	"github.com/openshift/ci-tools/pkg/junit"
+)
+
+// The TestRunSchema below is used to build the "TestRuns" table.
```

Comment:
Per other PR this should move down to right above the TestRunSchema variable, and read "TestRunSchema is used to build the TestRuns table."

Reply:
Suggest renaming to match the table, pluralizing Run(s)